.changeset/feat-charts-add-new-charts.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-docs': patch
```

Comment:
We need this to mention the charts package so they get released on the next release. So lets update this to say `minor`